lib/between_meals/changes/cookbook.rb

```diff
@@ -67,27 +119,33 @@ def initialize(files, cookbook_dirs)
 
       # Given a list of changed files
       # create a list of Cookbook objects
-      def self.find(list, cookbook_dirs, logger)
+      def self.find(list, cookbook_dirs, logger, repo, track_symlinks = false)
         @@logger = logger
         return [] if list.nil? || list.empty?
         # rubocop:disable MultilineBlockChain
+        @repo_dir = File.realpath(repo.repo_path)
+        @cookbook_dirs = cookbook_dirs
+        @track_symlinks = track_symlinks
+        list += symlinks(list) if @track_symlinks
```

Comment:
would it make more sense to do the 'append a `/`' here so that the Cookbook object doesn't need to know about track_symlinks at all?